web-app/src/tasks/tasks.py

```diff
@@ -15,6 +15,9 @@
 
 logger = get_task_logger(__name__)
 
+CONF_THRESHOLD=0.5
+MAX_LOOKBACK=60
```

Comment:
Something is still not right with keypoints extraction for pony vs killjoys. Before I bump up the `max_lookback` again, I need to double check if there aren't other bugs.